include/cantera/transport/HighPressureGasTransport.h

```diff
@@ -66,26 +112,550 @@ class HighPressureGasTransport : public MultiTransport
+     * @param Tr  Reduced temperature
+     * @param MW  Molecular weight
+     * @return double
+     */
+    double FQ_i(double Q, double Tr, double MW);
+
+    /**
+     * @brief  Returns the polarity correction term for a species based on reduced
+     * temperature, reduced dipole moment, and critical compressibility. Used in
+     * the viscosity calculation.
+     *
+     * Calculates polarity correction term of the Lucas method for a species based
+     * on the reduced temperature(Tr) and molecular weight(MW). Equation 9.4.18.
+     *
+     * @f[
+     *  \begin{equation}
+     *   F_P^0 =
+     *   \begin{cases}
+     *       1 & 0 \leq \mu_r < 0.022 \\
+     *      1 + 30.55(0.292 - Z_c)^{1.72} & 0.022 \leq \mu_r < 0.075 \\
+     *      1 + 30.55(0.292 - Z_c)^{1.72} \times 0.96 + 0.1(T_r - 0.7) & 0.075 \leq \mu_r
+     *   \end{cases}
+     *  \end{equation}
+     *
+
+     * @note The original description in Poling(2001) neglects to mention what happens
+     * when the quantity raised to the 1.72 power goes negative. That is an undefined
+     * operation that generates real+imaginary numbers. For now, we
+     * take the absolute value of the argument.
+     *
+     * @param mu_r  Species Reduced dipole moment
+     * @param Tr  Reduced temperature
+     * @param Z_c  Species Critical compressibility
+     * @return double
+     */
+    double FP_i(double mu_r, double Tr, double Z_c);
+
+};
+
+
+//These are the parameters that are needed to calculate the viscosity using the Chung method.
+struct ChungMixtureParameters
+{
+    // Mixture critical properties used by the Chung viscosity model.
+    double Vc_mix = 0;
+    double Tc_mix = 0;
+
+    // Values associated with the calculation of sigma and the molecular weight used in the Chung viscosity model.
+    double sigma_mix = 0;
+    double epsilon_over_k_mix = 0;
+    double MW_mix = 0;
+
+    // Values associated with the calculation of the Fc factor in the Chung viscosity model.
+    double mu_mix = 0;
+    double mu_r_mix = 0;
+    double acentric_factor_mix = 0;
+    double kappa_mix = 0;
+};
+
+//! Class ChungHighPressureTransport implements transport properties for
+//! high pressure gas mixtures.
+/*!
+ * The implementation employs a method of corresponding states, using the Takahashi
+ * @cite takahashi1975 approach for binary diffusion coefficients (using mixture
+ * averaging rules for the mixture properties), and the Chung method for the viscosity
+ * and thermal conductivity of a high-pressure gas mixture. All methods are described
+ * in Poling et al. @cite poling2001 (viscosity in Ch. 9, thermal conductivity in
+ * Ch. 10, and diffusion coefficients in Ch. 11).
+ *
+ * @note: All equations that are cited in this implementation are from the 5th edition
+ * of the book "The Properties of Gases and Liquids" by Poling, Prausnitz, and O'Connell.
+ *
+ * @ingroup tranprops
+ */
+class ChungHighPressureGasTransport : public MixTransport
+{
+protected:
```

Comment:
I think it would make sense to move this declaration into a separate `.h` file (and likewise, the implementation for this class into its own `.cpp` file)